src/View/Requirements_Backend.php

```diff
@@ -52,6 +53,20 @@ class Requirements_Backend
      */
     private static $combine_in_dev = false;
 
+    /**
+     * Determine if relative urls in the combined files should be converted to absolute.
+     *
+     * By default combined files will be parsed for relative URLs to image/font assets and those
+     * URLs will be changed to absolute to accomodate the fact that the combined css is placed
+     * in a totally different folder than the source css files.
+     *
+     * Turn this off if you see some unexpected results.
+     *
+     * @config
+     * @var bool
+     */
+    private static $resolve_relative_css_refs = true;
```

Comment:
```suggestion
    private static $resolve_relative_css_refs = false;
```
I think we'll have to have this `false` by default to avoid breaking people's sites - I can imagine situations where people have manually set their relative paths to be relative to the combined file location, which this would break.

Once this PR is merged, it would be good to create a new PR targetting `5` setting this to true.

Reply:
Understood. Will do that. Thank you.